crates/modelardb_server/src/storage/data_transfer.rs

```diff
@@ -231,6 +236,14 @@ impl DataTransfer {
             transferred_bytes, path,
         );
 
+        // If the server was started with a manager, transfer the metadata of the transferred file.
+        if let ClusterMode::MultiNode(manager) = &self.cluster_mode {
```

Comment:
Is it necessary to perform this check? Is it possible to transfer files to an object store without a manager?

Reply:
Good catch, it also made me find and fix a bug where you could update the remote object store on an edge without a manager which should not be possible.

Reply:
However, since we now always transfer metadata when transferring, I have had to remove all the data transfer component tests that transfer data. When we add testing for the manager later, I think most of these tests can be added back as integration tests.

Reply:
Ok, maybe create a task to add these tests back in to the code base as integration tests with a link to this discussion and the tests that you had to remove.

Reply:
Fixed.